hieradata/role/jenkins-slave.yaml

```diff
@@ -3,13 +3,18 @@ psick::pre::windows_classes:
   chocolatey: chocolatey
 psick::base::windows_classes:
   jenkins_slave: profile::ssh_windows
+  dns: profile::route53
```

Comment:
Just wondering: This approach of including a class that create the route53 record for a node is nice, but why only on jenkins-slave and not all the nodes?

Reply:
The rest of the nodes are registered with Terraform on lauching